test/integration/oracle/ChainlinkCompositeOracleArbitrum.t.sol

```diff
@@ -40,7 +40,11 @@ contract ChainlinkCompositeOracleArbitrumTest is Test {
             wstethstEthOracle
         );
 
-        vm.rollFork(102516073);
+        /// this needs to be updated every 6 months for tests to pass as rolling
+        /// to blocks too far in the past will cause the test to fail due to the rpc provider
+        /// updating this value to the current block number means tests will fail if the eth
+        /// price changes, so those will need to be updated too.
+        vm.rollFork(197234344);
```

Comment:
wouldn't be better to run this integration test against a hardcoded block.number instead?